scrapy_autounit/utils.py

```diff
@@ -133,6 +138,11 @@ def unpickle_data(data, encoding):
 
 def response_to_dict(response):
     return {
+        'cls': '{}.{}'.format(
+            type(response).__module__,
+            getattr(type(response), '__qualname__', None) or
+            getattr(type(response), '__name__', None)
```

Comment:
`__name__` isn't a substitute for `__qualname__` but as long as it isn't a nested class it should be fine -- all the Scrapy-provided classes are module top-level so there shouldn't be any problem, and Python 2 support is nearing sunset.